README.md

```diff
@@ -12,13 +12,15 @@ Generally, when we speak about React we talk about both [React](https://www.npmj
 
 **ReactDOM** is the glue between React and the [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model). When you want to show your React application you need to use `ReactDOM.render()` from the ReactDOM package. This package include the [reconciliation algorithm](#reconciliation) and platform-specific code – also known as [renderers](#renderers).
 
-**React** – often referred to as React core and includes [the top-level React APIs](https://reactjs.org/docs/react-api.html#react). It only includes the APIs necessary to define components: the component base class, lifecycle methods, state, props and all the concepts we know and love.
+**React** – often referred to as React core and includes [the top-level React APIs](https://reactjs
+.org/docs/react-api.html#react). It only includes the APIs necessary to define components: the component base class, 
+lifecycle functions, state, props and all the concepts we know and love.
 
 ## <a name="react-elements"></a> React elements
 
 React elements are the building blocks of React applications. React elements might be confused with the concept of
 React components. To clarify, React elements are generally what gets rendered on the screen, i.e. the return value of
-the `render()` method of a React component or the return of a functional component.
+the `render()` function of a React component or the return of a functional component.
```

Comment:
Hvorfor endre dette? Det er en `method`

Reply:
"A method is similar to a function, but is internal to part of a class."

Reply:
Men alt er funksjoner i JavaScript 😎  -> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes

Reply:
Extend the `render` function of the `VDomNode` function with a function that.... 😏

Reply:
a function**al**... 😏 

Pluss vi har en lett blanding, du endret nettopp til function og [fjernet metode](https://github.com/sveinpg/build-your-own-react/pull/22/files#diff-04c6e90faac2675aa89e2176d2eec7d8R274). Vi bør være konsekvente, hvis vi skal bruke `metode` må det isåfall være pga at det er en forskjell fra en funksjon, men ikke for noe som er det samme.

Reply:
Jeg er egentlig enig at klasser har metoder, ikke funksjoner.
Men det viktigste er klart at vi er konsekvente, så vi går for én av dem 😄

Reply:
Ok, går for function enn så lenge. Så kan vi se etterhvert. Til nå så har vi hvertfall brukt det mye om hverandre uansett om det har vært en klasseinstans eller ikke. 😉